tests/mock/v2/project/APIKEY/connect

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This is the same as `connect.json`, wondering why you need both?

Reply:
One is to test the underlying client, one is to test the user client. They use different methods for retrieving responses so we have to duplicate them